docassemble/ALWeaver/interview_generator.py

```diff
@@ -1066,6 +1070,11 @@ def escape_quotes(text: str) -> str:
     return text.replace('"', '\\"').replace("'", "\\'")
 
 
+def escape_double_quoted_yaml(text: str) -> str:
+    """Escape only double quotes in a string and the escape character itself"""
+    return text.replace(r"\",r"\\").replace('"', r"\"")
```

Comment:
```suggestion
    return text.replace("\\",r"\\").replace('"', r"\"")
```
It's silly, but you can't end a string (even a raw string) with a backslash, as it will escape the last quote.